wpilibc/Athena/src/PWM.cpp

```diff
@@ -100,6 +101,8 @@ void PWM::SetBounds(int32_t max, int32_t deadbandMax, int32_t center,
   m_centerPwm = center;
   m_deadbandMinPwm = deadbandMin;
```

Comment:
Actually apparently a pointer cast is just a compile time thing, so I'll just change the current implementation and remove this.